### DISCUSSION TITLE
Kata 10

### DISCUSSION THREAD

**Discussion by erick129n** [LaunchX-InnovaccionVirtual/CursoIntroPython#327] [Q&A] [answered]
hola compañeros, tengo un problema que aun no he conseguido resolver, es cuando ejecuto:
![image](https://user-images.githubusercontent.com/95593792/154866293-19313cf3-b7e0-4476-87ce-3062f8cd1e75.png)

teniendo en cuenta que tengo el directorio creado al ejecutarlo me sale esto:
![image](https://user-images.githubusercontent.com/95593792/154866333-8151fc50-a338-4eec-b252-e9305e8ab479.png)

y no sale como indican las instrucciones:
![image](https://user-images.githubusercontent.com/95593792/154866359-858a1460-454a-4ca1-9a30-2d30611ffba6.png)

seria de mucha ayuda que me resolvieran este problema, ya tengo mucho tratando de resolverlo pero nada

**Comment by Adagiominor** [comment, marked as answer]
Lo que te sale en el ejemplo/instrucciones del modulo 10 , es cuando no hay un control de excepciones definidas en codigo. Es decir, cuando no usas "try" y "except" en el archivo .py, si borras las condiciones y dejas la instruccion "open ('config.txt')" sola y justo debajo del 'main()', se ejecutara la instruccion. Como 'config.txt' no existe, te dara el error del ejemplo.

**Reply by erick129n** [reply]
haa, muchisimas gracias
